compiler-core/src/javascript/import.rs

```diff
@@ -92,9 +92,16 @@ impl<'a> Import<'a> {
 
     pub fn into_doc(self) -> Document<'a> {
         let path = Document::String(self.path.clone());
+        let import_modifier = if self.path.ends_with(".d.ts") {
```

Comment:
Could we pass this in as an argument or other form of configuration rather than checking the path please.

Reply:
Initially I thought about adding an argument but I don't know Rust or this code base so went with a change of least surface level. I'll look into converting it into an argument.

Reply:
Added boolean `is_type` property to `Import`. Can rename if there are better suggestions.